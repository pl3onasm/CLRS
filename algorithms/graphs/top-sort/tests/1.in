6 1
0 2
2 3
1 5
0 1
5 4
5 3
1 2
6 5

