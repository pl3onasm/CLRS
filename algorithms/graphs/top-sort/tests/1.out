TOPOLOGICAL SORT
----------------
0
6
1
2
5
4
3